PowerControl/Helpers/ProfileSettings.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using CommonHelpers;
+
+namespace PowerControl.Helper
+{
+    public class ProfileSettings : BaseSettings
+    {
+        private static string profilesPath = "Profiles";
+
+        static ProfileSettings()
+        {
+            Directory.CreateDirectory(profilesPath);
```

Comment:
This is not best, since it assumes that CWD equals to the current process directory. Which is mostly true, but not always.